docs/src/pages.ts

```diff
@@ -325,12 +329,13 @@ const pages: readonly MuiPage[] = [
   },
   {
     pathname: '/styles',
-    title: 'Styles (legacy)',
+    title: 'Styles',
     icon: 'StyleIcon',
+    legacy: true,
     children: [
-      { pathname: '/styles/basics' },
-      { pathname: '/styles/advanced' },
-      { pathname: '/styles/api', title: 'API' },
+      { pathname: '/styles/basics', legacy: true },
```

Comment:
Woudo it be enough like this? 

```suggestion
      { pathname: '/styles/basics' },
```

The badge draws the attention. I wonder if having it on one item is enough to pass the message.

Reply:
> Trying to make it more obvious that some pages are regarding a legacy API.

Did we try displaying the badge in the H1 instead of each sub menu item?

Reply:
I've added (LEGACY) suffix on the h1 on the pages (didn't really manage to display the badge itself) and removed the badges from the pages, leaving it only on the Styles sidebar item. This is now it looks:

![image](https://user-images.githubusercontent.com/4512430/153570363-42c6cc41-4275-44d2-bcb8-b8ea25eea9b2.png)

Reply:
I'm good with it, great work :)